_dprhtml/features/installation/init.js

```diff
@@ -32,6 +32,19 @@ var DPRComponentRegistry = (function () {
         return DPR_G.dtUrlsToPrefetch.map(x => `${DPR_Translations.trProps.dt.baseUrl}/${x}`)
       },
     },
+    {
+      id: 'ati',
+      name: 'Access to Insight',
+      shortDescription: '',
+      routeRegExp: /digitalpalireader\.online\/ati\/tipitaka/i,
+      isAvailable: () => true,
+      type: componentTypeTranslation,
+      sizeMB: 6,
+      getFileList: async () => {
+        await DPR_PAL.addOneJS('/_dprhtml/js/ati_list.js')
```

Comment:
We'll have to add a seperate list like how you added for bt. Reason is this way we won't downloads the css etc meaning formatting will be messed up offline.

Reply:
Please also add my, th, en, sa and delete the commented out code from install handler in sw.js

The pic I put on discord was only for demo purposes. I haven't made the changes.